packages/wdio-sync/src/wrapCommand.js

```diff
@@ -35,9 +35,8 @@ export default function wrapCommand (commandName, fn) {
         }
         /**
          * all named nested functions run in parent Fiber context
-         * except of debug and waitUntil
          */
-        this._NOT_FIBER = fn.name !== '' && fn.name !== 'debug' && commandName !== 'waitUntil'
+        this._NOT_FIBER = fn.name !== ''
```

Comment:
What is the reason behind this change?

Reply:
It's enough to delete flag here https://github.com/mgrybyk/webdriverio/blob/f5ae88d85aa71afb29d274dfd02c98c0b14a3c7f/packages/wdio-sync/src/runFnInFiberContext.js to make waitUntil, debug and other functions that use runFnInFiberContext work properly.
Ideally I'd not rely on function name at all, but I don't have a solution right now.